DESCRIPTION

```diff
@@ -14,8 +14,7 @@ Encoding: UTF-8
 LazyData: true
 Imports:
   data.table (>= 1.11.4),
-  foreach (>= 1.4.4),
-  locfit (>= 1.5-9.1)
```

Comment:
The locfit package has to stay. We don't explicitly call it, but if the user doesn't have it, the dispersion function doesn't work.